site/src/components/LicenseCard/LicenseCard.tsx

```diff
@@ -30,6 +30,9 @@ export const LicenseCard = ({
     number | undefined
   >(undefined)
 
+  const currentUserLimit =
```

Comment:
Should we add some unit tests to make sure that we present the correct licensing info?

Reply:
fixed.